lib/schema.js

```diff
@@ -237,27 +284,47 @@ class Metaschema {
           value = def(value);
           obj[field] = value;
         } else {
-          value = this.createInstance(def, value);
+          if (def.category) value = this.createInstance(def.category, value);
```

Comment:
It should be possible to save an objects `id` instead of the whole object IMHO.

Reply:
So, from this schema:
```javascript
{
  field: { category: 'CategoryName' }
}
```
We can create such instance:
```javascript
{ 
  field: 1, // an id
}
```
?

Reply:
It should be possible to create: `{ field: new Id(1) }` from: `{ field: 1 }`.

Reply:
What is an `Id` here. Is it included in `@metarhia/common` or I should define it by myself?

Reply:
You may use `Int64` for now.